rake_config/vanishing_data.rb

```diff
@@ -0,0 +1,74 @@
+class SourceCSV
+  def initialize(str)
+    @raw = str
+  end
+
+  def ids
+    grouped.keys
+  end
+
+  def rows(*kk)
+    kk.map { |k| grouped[k].first }
```

Comment:
Doesn't taking the `.first` item from the group mean we might potentially loose data if a membership CSV has multiple rows for a single member with the same id?